### ISSUE TITLE
Blocked dates

### ISSUE THREAD

**Issue by leonardobork** [bibekg/react-schedule-selector#23]
Add the option to block dates and times (like, if you aren't available at the selected time)

**Comment by bibekg** [comment]
Thanks for filing the issue! If I'm understanding correctly, are you suggesting that each cell should essentially have 3 states: unselected, selected positively, selected negatively (i.e. "blocked out" to specify not available)?

**Comment by ultimaustin** [comment]
Hi, This feature would be extremely useful. This project looks perfect for a requirement I have for self-scheduling but I need the ability to mark certain slots in the calendar as blocked off. @bibekg I was hoping that there was a mechanism to be able to set the state of a range of times as "selected negatively" so they would be greyed out and un-selectable. 

Am I right that this is currently not supported?

**Comment by bibekg** [comment]
@ultimaustin yeah, that isn't supported right now. But it does sound like you and @leonardobork may be describing different features.

I think what @leonardobork is requesting is to be able to allow _the user_ to negatively select a cell to indicate they are not available at that time.

It sounds like your request is to _programmatically disable_ certain cells from being selected at all. 

Am I understanding correctly?

**Comment by ultimaustin** [comment]
Yes, you are completely right.

I am requesting the ability to prevent the user from selecting certain cells. For any given day I know which cells are available to book and which have already been booked. I therefore need to be able to prevent the user from selecting the slots which have already been chosen by someone else.

**Comment by bibekg** [comment]
@ultimaustin Gotcha. That seems like a pretty useful feature, happy to add it in the next release. Would you mind filing a new issue describing it to reduce confusion with the other feature that I think is being described in this one?
